planex/spec.py

```diff
@@ -560,10 +560,16 @@ def specpath(self):
         """Return the path to the spec file"""
         return self.path
 
-    def rewrite_spec(self):
+    # pylint: disable=too-many-locals
+    def rewrite_spec(self, srpm_sources=None, manifests=None):
         """
         Rewrite the sources and patches in the spec file, also inserting the
-        names of all patchqueue patches into it
+        names of all patchqueue patches into it.
+
+        If [srpm_sources] is a list of sources, it will add comments in the
+        spec file describing the origin of the source.
+        If [manifests] is true it will add `Provides: gitsha(sha) = url` using
```

Comment:
Hopefully this is not what happens, should be `Provides: gitsha(URI) = SHA`

Reply:
It is :P I'll update the code and fix the docstring